DIPs/DIP-32.md

```diff
@@ -0,0 +1,112 @@
+
+## Motivation & Rationale
+
+### Giving people private data to selectively reveal with ZK
+
+ZK as a technology is rapidly progressing on many fronts: prover speed, recursive verification, and developer friendliness. But for users to make meaningful proofs of identity, they need a store of personal cryptographic data. Unfortunately, most of our data is locked up in the databases of big tech companies, which is only meaningful on their respective platforms. And even if we could access our data, it would be meaningless without some sort of signature, as we have no way of verifying if it’s “real” or just generated by something like ChatGPT.
+
+We believe NFC cards that produce unique signatures on every tap are a great solution to this initial “signed data bootstrapping” problem, and are well suited to capture in-person interactions: attending a talk, meeting a friend, going to a concert, buying your 4th coffee at a cafe, etc. And because these signatures come directly from the NFC card, your collection of signatures is completely private to your phone, and can be selectively revealed with ZK.
+
+### Generate understanding of ZK through an interactive experience
+
+Past not having personal data to make proofs about, actually making the proof also feels out of reach for most people. This could be due to perceptions that generating a proof requires a technical background or is too computationally intensive for personal devices. We want to change that perception by allowing everyone at Devconnect to generate a gamified, unserious proof: the rarity of your Jubjub collection, without revealing the specific set of Jubjubs you've collected.
+
+We are hoping that once folks get to experience private data ownership and making their own proofs, they will be able to more actively ideate on how their communities can benefit from private data ownership and selective revealing via ZK.
+
+### Making Devconnect more fun, interactive, and collaborative
+
+We hope people will explore Devconnect and Istanbul more in an attempt to collect these Jubjubs and compete for their team. We also hope by placing everyone in random teams, attendees will be able to connect with fellow teammates over the game, especially thouse who otherwise would never meet at the conference.
```

Comment:
```suggestion
We hope people will explore Devconnect and Istanbul more in an attempt to collect these Jubjubs and compete for their team. We also hope by placing everyone in random teams, attendees will be able to connect with fellow teammates over the game, especially those who otherwise would never meet at the conference.
```